pkg/nodeinfo/attributes.go

```diff
@@ -91,8 +98,9 @@ func newNodeAttributes(node *corev1.Node) NodeAttributes {
 	nLabels := node.GetLabels()
 	for attrType, label := range attrToLabel {
 		err = attr.fromLabel(AttributeType(attrType), nLabels, label)
-		if err != nil {
-			log.V(consts.LogLevelWarning).Info("Warning:", "cannot create NodeAttribute(%x), %v", attrType, err)
+		if err != nil && attrType < OptionalAttrsStart {
```

Comment:
Do we need this change? We'll miss logs if operator fails to create optional node attribute

Reply:
LGTM, thanks for the clarification